backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/UpdateVmCommand.java

```diff
@@ -948,6 +948,10 @@ private void updateVmNumaNodes() {
     }
 
     private void initNuma() {
+        if (VmCpuCountHelper.isResizeAndPinPolicy(getParameters().getVm())) {
+            getParameters().getVm().setvNumaNodeList(new ArrayList<>());
+            return;
+        }
```

Comment:
This part overcomes many flows. but the best would be to initially not have the NUMA nodes.
For example: when editing in the UI, have the `vNumaNodeList` as empty array. I think PR #488 might need to merge first and then we can tackle this part. @ljelinkova

Reply:
that part is debatable, I'd expect to see something like setUpdateNuma(false) if it's not handled on the UI side rather than replacing the list with an empty list

Reply:
I think Liran is correct here - if the resize and pin is selected, we should set the empty list. With Arik's suggestion, if the numa was previously set and the user would then select resize and pin, the original setting would not be affected and the VM would still have numa nodes.

Reply:
This won't be enough. We do need to clear it on each update.
The reason taken from the bug:
1. The VM is created with 4 CPUs, 'Resize_and_pin_numa' policy.  DON'T pin to a host and Run_once VM on a specific host.
2. Try to pin such VM to a host in Edit/Host window.
Editing a VM that is set as resize and pin, will hit an error, since we have static NUMA configuration.
If we do it only when switching to resize (which we don't really care, we override it anyway) later on, when the VM is running we will hit errors.

Reply:
looks better now

Reply:
done